fetch_make.py

```diff
@@ -11,27 +11,49 @@
+    for option in ['--headless','--disable-gpu','--window-size=1920,1200','--ignore-certificate-errors','--disable-extensions','--no-sandbox','--disable-dev-shm-usage']:
+        chrome_options.add_argument(option)
+    global chromedriver
+    chromedriver = webdriver.Chrome(service = chrome_service,options = chrome_options)
 
-driver.get('https://voyager.jpl.nasa.gov/mission/status/')
-page_source = driver.page_source
+def fetch_data(url,need_selenium = False):
+    if need_selenium:
+        chromedriver.get(url)
+        return chromedriver.page_source
+    else:
+        response = get(url)
+        return response.text
 
-sections = ['km','au','kms','aus','speed','lt']
-AllUp_content = '<includeonly>{{#switch:{{{1|}}}|t=time|1={{#switch:{{{id}}}|1={{#switch:{{{section}}}|km=data1 |au=data2 |kms=data3 |aus=data4 |speed=data5 |lt=data6 |#default=请输入正确的选项名！}}|2={{#switch:{{{section}}}|km=data7 |au=data8 |kms=data9 |aus=data10 |speed=data11 |lt=data12 |#default=请输入正确的选项名！}}|#default=请输入正确的编号！}}|#default=请输入正确的AllUp编号！}}</includeonly><noinclude>[[Category:模板]]{{documentation}}</noinclude>'.replace('time',strftime('%Y年%m月%d日%H时',localtime()))
-cnt = 0
-for id in [1,2]:
-    for section in sections:
-        cnt += 1
-        data_re = compile(f'id="voy{id}_{section}">(.*)</div>')
-        findall_res = findall(data_re,page_source)
-        AllUp_content = AllUp_content.replace(f'data{cnt} ',f'{findall_res[0]}')
+def make(id):
+    match id:
+        case 't':
+            return strftime('%Y年%m月%d日%H时',localtime())
+        case 1:
+            web_data = fetch_data(
+                'https://voyager.jpl.nasa.gov/mission/status/',
+                True
+            )
+            data = {}
+            for id in [1,2]:
+                data[id] = {}
+                for section in ['km','au','kms','aus','speed','lt']:
+                    data[id][section] = findall(
+                        compile(f'id="voy{id}_{section}">(.*)</div>'),
+                        web_data
+                    )[0]
+            return f"{{{{#switch:{{{{{{id|}}}}}}|1={{{{#switch:{{{{{{section|}}}}}}|km={data[1]['km']}|au={data[1]['au']}|kms={data[1]['kms']}|aus={data[1]['aus']}|speed={data[1]['speed']}|lt={data[1]['lt']}|#default=请输入正确的选项名！}}}}|2={{{{#switch:{{{{{{section|}}}}}}|km={data[2]['km']}|au={data[2]['au']}|kms={data[2]['kms']}|aus={data[2]['aus']}|speed={data[2]['speed']}|lt={data[2]['lt']}|#default=请输入正确的选项名！}}}}|#default=请输入正确的编号！}}}}"
+        case 2:
+            return '敬请期待！'
+        case _:
+            return '请输入正确的AllUp编号！'
 
-system(f'echo "ALLUP_CONTENT={AllUp_content}" >> $GITHUB_OUTPUT')
-
-print(f'==== AllUp_Content ====\n{AllUp_content}')
\ No newline at end of file
+if __name__ == '__main__':
+    configure_chromedriver()
+    AllUp_content = f"<includeonly>{{{{#switch:{{{{{{1|}}}|t={make('t')}|1={make(1)}|2={make(2)}|#default={make('default')}}}</includeonly><noinclude>[[Category:模板]]{{{{documentation}}</noinclude>"
```

Comment:
The correct wikitext should be: 
```go
<includeonly>{{#switch:{{{1|}}}|t={make('t')}|1={make(1)}|2={make(2)}|#default={make('default')}}}</includeonly><noinclude>[[Category:模板]]{{documentation}}</noinclude>
```

So the code should be: 
```suggestion
    AllUp_content = f"<includeonly>{{{{#switch:{{{{{{1|}}}}}}|t={make('t')}|1={make(1)}|2={make(2)}|#default={make('default')}}}}}</includeonly><noinclude>[[Category:模板]]{{{{documentation}}}}</noinclude>"
```